README.md

```diff
@@ -222,7 +231,13 @@ This command lets you display all the courses you have put in the Google Calenda
 #### 3.10 /isfree
 This command lets you check whether you are free at a certain time interval or not. 
 
-#### 3.11 /quit
+#### 3.11 /getupcomingevent
+This command displays all your upcoming events
+
+#### 3.12 /help
+This command displays all the commands present in the bot
+
+#### 3.13 /quit
 This quits the bot.
 
 ## Chapter 4 Inputing Course Indexes and Finding Common Free Time
```

Comment:
Is the word **inputing** grammatically correct? hahah sorry for being neat-picky.
This is a readme file, so I think we should pay attention to the grammar as well.